EtherealCereal.xcodeproj/xcshareddata/xcschemes/Tests.xcscheme

```diff
@@ -1,58 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
deleted tests scheme ?

Reply:
Built inside the framework scheme instead. 🚀